wcfsetup/install/files/acp/templates/index.tpl

```diff
@@ -136,7 +136,7 @@
 			<dl>
 				<dt>{lang}wcf.acp.index.system.php.version{/lang}</dt>
 				<dd>
-					{if $__wcf->session->getPermission('admin.configuration.package.canInstallPackage') && $__wcf->session->getPermission('admin.configuration.package.canUpdatePackage')}
+					{if (!ENABLE_ENTERPRISE_MODE || $__wcf->getUser()->hasOwnerAccess()) && $__wcf->session->getPermission('admin.configuration.package.canInstallPackage') && $__wcf->session->getPermission('admin.configuration.package.canUpdatePackage')}
```

Comment:
Should this information even be available in Enterprise mode? So PHP version, server load, etc.

Reply:
I agree, none of this is relevant for the "average" user. Please remove the "Server" and "PHP" section using the same condition as in the change above.